flink-end-to-end-tests/test-scripts/test_streaming_elasticsearch.sh

```diff
@@ -39,10 +39,31 @@ on_exit test_cleanup
 TEST_ES_JAR=${END_TO_END_DIR}/flink-elasticsearch${ELASTICSEARCH_VERSION}-test/target/Elasticsearch${ELASTICSEARCH_VERSION}SinkExample.jar
 
 # run the Flink job
-$FLINK_DIR/bin/flink run -p 1 $TEST_ES_JAR \
+JOB_ID=$($FLINK_DIR/bin/flink run -d -p 1 $TEST_ES_JAR \
   --numRecords 20 \
   --index index \
-  --type type
+  --type type | awk '{print $NF}' | tail -n 1)
 
+
+# wait for 10 seconds
+wait_job_submitted ${JOB_ID}
```

Comment:
isn't the loop below automatically handling this case as well?

Reply:
This call just waits for the job to be submitted (somehow). The second loop expects the job to be in state "RUNNING".
It would fail if the job status was "CREATED", which I assume is a state that we might see.

To be honest, I can also undo the changes to the elasticsearch script + the common.sh script. I just left them in in case the tests are failing in the future.

With my changes, the tests will at least fail after a few seconds, instead of hanging indefinitely, and they will print the Flink logs to make debugging easier.

Reply:
sounds like we can simplify this by checking whether the job has finished instead.

Reply:
I will undo my changes. I don't spend more time with this.